Content/paket.dependencies

```diff
@@ -64,4 +64,7 @@ group Build
     nuget Fake.IO.Zip
     github CompositionalIT/fshelpers src/FsHelpers/ArmHelper/ArmHelper.fs
 //#endif
+//#if (deploy == "gcp-kubernetes")
+    nuget Fake.Tools.Git
```

Comment:
we'll need to add new case to paket lock generation for this - I can add that afterwards

Reply:
sounds good, since I have no clue about how that part works.